hedera-dependency-versions/build.gradle.kts

```diff
@@ -28,6 +28,15 @@ dependencies {
 }
 
 dependencies.constraints {
+    api("io.helidon.common:helidon-common:4.1.1") {
+        because("io.helidon.common")
+    }
+    api("io.helidon.webclient:helidon-webclient:4.1.1") {
+        because("io.helidon.webclient")
+    }
+    api("io.helidon.webclient:helidon-webclient-grpc:4.1.1") {
+        because("io.helidon.webclient.grpc")
+    }
```

Comment:
The use of protoc objects and the helidon-grpc libraries (with secondary grpc.io dependencies) _may not_ be necessary.
The PBJ library has a [helidon "plugin"](https://github.com/hashgraph/pbj/tree/main/pbj-core/pbj-grpc-helidon) to replace the helidon grpc plugin that is _nearly_ complete (modulo a [few bug issues](https://github.com/hashgraph/pbj/issues/306)).

It may be worth evaluating the status of that work to determine if a slight delay of this work would enable use of that plugin to avoid introducing protoc-compiled classes and these helidon grpc dependencies.

Reply:
Initially, this issue was meant to be a "first pass" API that will give us the possibility to iterate over it and improve it down the line. But this will be something to consider, for certain. Thanks!